dependency/dependency_test.go

```diff
@@ -239,6 +229,34 @@ func (c *ClientSet) createConsulTestResources() error {
 	return nil
 }
 
+func (c *ClientSet) createConsulSamenessGroups(name, partition, failoverPartition string) error {
+	members := append([]api.SamenessGroupMember{}, api.SamenessGroupMember{
+		Partition: partition,
+	}, api.SamenessGroupMember{
+		Partition: failoverPartition,
+	})
+	sg := &api.SamenessGroupConfigEntry{
+		Kind:               QuerySamenessGroup,
+		Name:               name,
+		Partition:          partition,
+		DefaultForFailover: true,
+		Members:            members,
+	}
+	log.Printf("sameness group: %+v", sg)
+	_, _, err := c.consul.client.ConfigEntries().Set(sg, &api.WriteOptions{})
+	if err != nil {
+		return err
+	}
+
+	sgs, _, err := c.consul.client.ConfigEntries().List(QuerySamenessGroup, &api.QueryOptions{})
```

Comment:
why do we need to list them at the end here?

Reply:
it was debugging for the long line below.  I am removing these remnants.

Reply:
removed in edd0f7394427f64c6fc9de7aa74dd6bac9c32f80